lib/perl/Genome/VariantReporting/Suite/Joinx/Dbsnp/DbsnpInterpreter.pm

```diff
@@ -43,17 +43,20 @@ sub _interpret_entry {
     my $caf = $parser->process_entry($entry);
     my %return_values;
 
+    my $has_tag = $entry->info($parser->TAG);
     for my $variant_allele (@$passed_alt_alleles) {
-        if (!defined $caf) {
```

Comment:
Was this case impossible?  If not then the new code will emit warnings in the two places `$caf->{$variant_allele}` is called.